### PR TITLE
Adjust the repository URLs

### PR DESCRIPTION
**Pull request by nickvergessen**
@LukasReschke

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,16 +11,15 @@ If you have questions about how to install or use ownCloud, please direct these
...
-[mailinglist]: https://mailman.owncloud.org/mailman/listinfo/owncloud
-[forum]: https://forum.owncloud.org/
-[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
+[template]: https://raw.github.com/nextcloud/server/master/issue_template.md
+[forum]: https://help.nextcloud.com/
+[irc]: https://webchat.freenode.net/?channels=nextcloud
 
 ### Contribute Code and translations
-Please check [core's contribution guidelines](https://github.com/owncloud/core/blob/master/CONTRIBUTING.md) for further information about contributing code and translations.
+Please check [core's contribution guidelines](https://github.com/nextcloud/core/blob/master/CONTRIBUTING.md) for further information about contributing code and translations.
```
Comment (nickvergessen) [inline CONTRIBUTING.md]: s/core/server

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,16 +11,15 @@ If you have questions about how to install or use ownCloud, please direct these
   - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
   - You can also filter by appending e. g. "state:open" to the search string.
   - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
-* This repository ([Notifications](https://github.com/owncloud/notifications/issues)) is *only* for issues within the ownCloud Announcement Center code.
-* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* This repository ([Notifications](https://github.com/nextcloud/notifications/issues)) is *only* for issues within the Nextcloud Notifications code.
+* __SECURITY__: Report any potential security bug to security@nextcloud.com following our [security policy](https://nextcloud.com/security/) instead of filing an issue in our bug tracker.
```
Comment (LukasReschke) [inline CONTRIBUTING.md]: Best point to HackerOne instead: hackerone.com/nextcloud

Comment (nickvergessen) [inline CONTRIBUTING.md]: k, I was not sure, same for activity I guess

**Comment by nickvergessen** [comment]
Fixed both comments

**Comment by MorrisJobke** [comment]
👍

**Comment by mar1u5** [comment]
👍

**Comment by mar1u5** [comment]
We should port that to `stable9` 👊 @nickvergessen

**Comment by nickvergessen** [comment]
Jup, will do on monday

**Pull request merged**